web/sites/default/settings.php

```diff
@@ -788,3 +788,8 @@
 $config_directories['sync'] = '../config/sync';
 
 $settings['install_profile'] = 'standard';
+
+// The sinch key.
+$settings['sinch-key'] = '1ece0175-849e-42d7-9b4b-b9fd1a1c1358';
+// The sinch secret key.
+$settings['sinch-secret'] = '8L/Eug038U+ino3ztJvGrA==';
```

Comment:
Secret key in a public repo? http://www.reactiongifs.com/r/panic1.gif  It's named SECRET for a reason.